invenio_theme_tugraz/config.py

```diff
@@ -49,6 +53,9 @@
 # Invenio-accounts
 # ============
 # See https://invenio-accounts.readthedocs.io/en/latest/configuration.html
+COVER_TEMPLATE = 'invenio_theme_tugraz/accounts/accounts_base.html'
```

Comment:
This file has conflict due to the current changes in the master branch, pull the changes from master first.